aws/src/main/python/updateDependencies/lambda_function.py

```diff
@@ -52,6 +52,10 @@
   - sudo apt-get install curl
   - curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
   - sudo apt-get install nodejs -y
+  - sudo apt-get install docker-ce docker-ce-cli containerd.io -y
```

Comment:
This list of things keeps growing. We already have a mechanism (lambda `updateDependencies`) to update our image. What if we change the way how we install all required libraries to be there, so it will be happening only one time when w update our AMI instead of running it for every single run.

Reply:
@nikolayilyin as discussed in person, I didn't pay an attention that it is already `updateDependencies`, so forget what I said :D